group_vars/matrix_servers

```diff
@@ -1552,11 +1552,11 @@ matrix_ssl_domains_to_obtain_certificates_for: |
   }}
 
 matrix_ssl_architecture: "{{
-	{
-		'amd64': 'amd64',
-		'arm32': 'arm32v6',
-		'arm64': 'arm64v8',
-	}[matrix_architecture]
+  {
+    'amd64': 'amd64',
+    'arm32': 'arm32v6',
+    'arm64': 'arm64v8',
+  }[matrix_architecture]
```

Comment:
Probably no need to change this?

Reply:
Oups yes, wrongly commited, I'll fix it 🙈🙈🙈

Reply:
Fixed 👍